projects/authentication/docs/user-access-token.md

```diff
@@ -39,11 +51,11 @@ sequenceDiagram
     autonumber
     actor User
     User-->>Client: Click login link
-    Client->>Auth server: Redirect to GET /authorize
+    Client->>Auth server: Redirect to GET /auth
     Auth server-->>User: Show login page
     User-->>Auth server: Login and give consent
     Auth server-->>Client: Redirect back to callback URL with authorization code
-    Client->>Auth server: POST /oauth/token with authorization code<br /> and client credentials
+    Client->>Auth server: POST /token with authorization code<br /> and client credentials
```

Comment:
```suggestion
    Client->>Auth server: POST /realms/uitid/protocol/openid-connect/token with authorization code<br /> and client credentials
```

Reply:
This is the mermaid diagram and I explicitly chose to leave out the full path. Maybe I can reduce confusion further by getting rid of the `/` 

This is how it looks like with the full path:
<img width="590" alt="image" src="https://github.com/user-attachments/assets/013648b7-2bbc-4c98-bd96-199120f04314">

Reply:
Changed to full path